frame/support/src/lib.rs

```diff
@@ -85,21 +85,26 @@ pub enum Never {}
 
 /// Create new implementations of the [`Get`](crate::traits::Get) trait.
 ///
-/// The so-called parameter type can be created in three different ways:
+/// The so-called parameter type can be created in four different ways:
 ///
-/// - Using `const` to create a parameter type that provides a `const` getter.
-///   It is required that the `value` is const.
+/// - Using `const` to create a parameter type that provides a `const` getter. It is required that
+///   the `value` is const.
 ///
 /// - Declare the parameter type without `const` to have more freedom when creating the value.
 ///
-/// - Using `storage` to create a storage parameter type. This type is special as it tries to
-///   load the value from the storage under a fixed key. If the value could not be found in the
-///   storage, the given default value will be returned. It is required that the value implements
-///   [`Encode`](codec::Encode) and [`Decode`](codec::Decode). The key for looking up the value
-///   in the storage is built using the following formular:
+/// - Using `storage` to create a storage parameter type. This type is special as it tries to load
+///   the value from the storage under a fixed key. If the value could not be found in the storage,
+///   the given default value will be returned. It is required that the value implements
+///   [`Encode`](codec::Encode) and [`Decode`](codec::Decode). The key for looking up the value in
+///   the storage is built using the following formula:
 ///
 ///   `twox_128(":" ++ NAME ++ ":")` where `NAME` is the name that is passed as type name.
 ///
+/// - Using `static` to create a struct that implements [`Get`](crate::traits::Get) and the value is
```

Comment:
Everything created by this macro implements this trait :P Meaning that you don't need to repeat yourself here. Just look at the top of the macro docs.